src/java.base/share/classes/jdk/internal/misc/CDS.java

```diff
@@ -333,6 +334,8 @@ private static void dumpSharedArchive(boolean isStatic, String fileName) throws
             throw new RuntimeException("Cannot rename temp file " + tempArchiveFileName + " to archive file" + archiveFileName);
         }
         // Everyting goes well, print out the file name.
-        System.out.println((isStatic ? "Static" : " Dynamic") + " dump to file " + cwd + File.separator + archiveFileName);
+        String archiveFilePath = cwd + File.separator + archiveFileName;
```

Comment:
`archiveFileName` may already be a absolute path (e.g., "/tmp/foo.jar"), so this will print out something like /app/cwd//tmp/foo.jar, which will be incorrect. It should be changed to

```
String archiveFilePath = new File(archiveFileName).getAbsolutePath();
```

Reply:
Yes, it could be a absolute path already.